casper/src/main/scala/coop/rchain/casper/util/rholang/RuntimeManager.scala

```diff
@@ -167,28 +168,34 @@ class RuntimeManagerImpl[F[_]: Concurrent] private[rholang] (
 
   def computeDeployPayment(start: StateHash)(user: ByteString, amount: Long): F[StateHash] =
     withResetRuntimeLock(start)(
-      computeDeployPayment(_)(user, amount).map(_.root.toByteString)
+      runtime =>
+        computeDeployPayment(runtime, runtime.reducer, runtime.space)(user, amount)
+          .map(_.root.toByteString)
     )
 
   private def computeDeployPayment(
-      runtime: Runtime[F]
+      runtime: Runtime[F],
```

Comment:
would it be possible to drop Runtime by just passing cost?

Reply:
Not until Joe gets rid of the error log ;/